README.md

```diff
@@ -1,70 +1,80 @@
-# Getting Started with Create React App
+# Todo-List App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+"Todo-List App" is a web application that allows you to organize your tasks. 
 
-## Available Scripts
+The present project is part of the Microverse curriculum Module 3 Block 2. The main objective is putting hands on practice to build a web application using React.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Html/CSS
+- JavaScript
+- React
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Additional Tools
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- Webhint
+- Stylelint
+- Eslint
+- Lighthouse
+- Webpack
 
-### `npm test`
+## Live Demo 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Live Demo App on Github-Pages](https://carshy.github.io/todos-react-application/)
 
-### `npm run build`
+## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+To get a local copy of this project:
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Clone this repository or download the Zip folder:
+```
+https://github.com/Carshy/math-magicians.git
+```
+ 
```

Comment:
- [ ] please consider putting the link for this project. The link above is for the Math magician project.